test/commands/genesis/genesis-manual-test.sh

```
@@ -186,7 +186,7 @@ $CLI genesis bucket add-unlocked "$GENESIS3" \
 echo "[20] Fetching unlocked bucket..."
 UNLOCK_OUT=$($CLI genesis bucket fetch "$GENESIS3" --bucketIndex 0 --type unlocked 2>&1)
 echo "$UNLOCK_OUT" | grep -q "Unlocked Bucket" && pass "Fetch unlocked bucket" || fail "Fetch unlocked bucket"
-echo "$UNLOCK_OUT" | grep -q "Claimed: No" && pass "Not yet claimed" || fail "Not yet claimed"
+echo "$UNLOCK_OUT" | grep -q "Amount Claimed: 0" && pass "Not yet claimed" || fail "Not yet claimed"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)